pip-requirements.txt

```diff
@@ -1,4 +1,4 @@
 cerberus
-data_tools @ git+ssh://git@github.com/tripactions/Data_Tooling.git@v1.9.6
+data_tools @ git+ssh://git@github.com/tripactions/Data_Tooling.git
```

Comment:
Be aware, since pip will be installing through git, this does not take the latest version but the latest commit in master, which could be somewhere in between versions.

Reply:
Nice catch @optroodt ! I'll actually pin it to 2.0 since I like this version :)